src/overlaybd/fs/zfile/crc32/crc32c.cpp

```diff
@@ -625,15 +626,43 @@ static uint32_t crc32c_dml(const uint8_t *data, size_t nbytes, uint32_t crc) {
     }
     return result.crc_value ^ 0xFFFFFFFF;
 }
-#endif
+
+static uint32_t crc32c_isal(const uint8_t *data, size_t nbytes, uint32_t crc) {
+    uint32_t sum = crc;
+
+    sum = crc32_iscsi((unsigned char *)data, nbytes, crc);
+    return sum;
+}
+
+bool check_dsa() {
+    struct pci_access *pacc;
+    struct pci_dev *dev;
+    unsigned int c;
+
+    pacc = pci_alloc();
+    pci_init(pacc);
+    pci_scan_bus(pacc);
+    for (dev = pacc->devices; dev; dev = dev->next) {
+        pci_fill_info(dev, PCI_FILL_IDENT | PCI_FILL_BASES);
+        if ((dev->vendor_id == 0x8086) && (dev->device_id == 0x0b25)) {
+            pci_cleanup(pacc);
+            return true;
+        }
+    }
+    std::cout << "No DSA found!" << std::endl;
```

Comment:
Suggest not print to stdout here. At least include alog.h and use LOG_INFO()

Reply:
ok